### PR TITLE
False positives and rules update

### LINKED ISSUE #645: Update false positive list
this is a false positive “best regards” \- in this situation, best should never be underlined\.

### PR DESCRIPTION
**Pull request by lnazarenko**
fixes https://github.com/witty-works/nlp_api/issues/645

- update US English rules
- update GB English rules
- update false positives English list
- update German rules

### CODE REVIEW

File: training_data/de-DE/abbreviations.csv [hunk 1]
```diff
@@ -145,7 +145,6 @@ KPIs,acr,style,abbreviation,"['Key Performance Indicators (KPI)', 'Leistungskenn
 Kptsb,acr,style,abbreviation,"['Komplexe posttraumatische Belastungsstörung (komplexe PTBS)', 'Komplexe posttraumatische Belastungsstörung (K-PTBS)']"
 LAMP,acr,style,abbreviation,"['Linux, Apache, MySQL, PHP (LAMP)', 'Linux, Apache, MySQL, Python (LAMP)', 'Linux, Apache, MySQL, Perl (LAMP)']"
 LBO,acr,style,abbreviation,"['Leveraged-Buyout (LBO)', 'fremdfinanzierte Übernahme']"
-LET,-acr,style,abbreviation,"['Leaving early today', 'Ich gehe heute früher', 'Muss heute früher los']"
```
Comment (lsmith77) [inline training_data/de-DE/abbreviations.csv]: @lnazarenko I think you need to update your script to include `-acr` in this file

Comment (lsmith77) [reply training_data/de-DE/abbreviations.csv]: they seem to all be moved to `style_words.csv`

File: training_data/en-GB/abbreviations.csv [hunk 2]
```diff
@@ -137,13 +138,12 @@ ISP,acr,style,abbreviation,"['Internet service provider', 'Internet service prov
...
 JS,acr,style,abbreviation,"['JavaScript', 'JavaScript (JS)']"
 JSON,acr,style,abbreviation,"['JavaScript object notation', 'JavaScript object notation (JSON)']"
-KISS,acr,style,abbreviation,"['Keep it simple, stupid', 'Keep it simple, stupid (KISS)', 'Keep it simple, obvious']"
+KISS,acr,style,abbreviation,"['Keep it simple, sweets', 'Keep it simple, stupid (KISS)', 'Keep it simple, obvious']"
 KPI,acr,style,abbreviation,"['key performance indicator', 'key performance indicators', 'key performance indicator (KPI)', 'key performance indicators (KPI)']"
 KPIs,acr,style,abbreviation,"['key performance indicators', 'key performance indicators (KPIs)']"
 LAMP,acr,style,abbreviation,"['Linux, Apache, MySQL, PHP', 'Linux, Apache, MySQL, PHP (LAMP)', 'Linux, Apache, MySQL, Python', 'Linux, Apache, MySQL, Python (LAMP)', 'Linux, Apache, MySQL, Perl', 'Linux, Apache, MySQL, Perl (LAMP)']"
 LBFM,abbr,openly_discriminating,sexism,['-']
 LBO,acr,style,abbreviation,"['leveraged buyout', 'leveraged buyout (LBO)']"
-LET,-acr,style,abbreviation,['Leaving early today']
```
Comment (lsmith77) [inline training_data/en-GB/abbreviations.csv]: @lnazarenko same issue here

File: training_data/en-GB/homonyms_words.csv [hunk 3]
```diff
@@ -1,34 +1,40 @@
 Lemma,Word_Type,Category,Primary_subcategory,Alt_split
-challenge,s,unconscious_bias,agentic,"['test', 'trial', 'burning question', 'complex situation', 'task that requires thought and skill to solve(...)', 'complex situations', 'difficult tasks', 'burning questions', 'trials']"
+Aryan,s,unconscious_bias,Nazi_language,"['-', 'Indo-Iranian', 'Indo-Aryan', 'Scandinavian looking person(...)', 'blond White person(...)', 'blond white person(...)', 'Indo-Iranians', 'Indo-Aryans']"
```
Comment (lsmith77) [inline training_data/en-GB/homonyms_words.csv]: @tje-ww subcategories should be lowercased. also the category should be added at least on https://www.notion.so/witty-works/e68e073dd0a342fca2a6683c7a8b2341?v=4edab194c6e9450eaa7c9b64409099f7

Comment (lnazarenko) [reply training_data/en-GB/homonyms_words.csv]: What should I do with Nazi_language subcategory? Should I keep these words (make subcategory lowercased) or remove and wait till Tracey will add it to the notion page and wrote description?

Comment (lsmith77) [reply training_data/en-GB/homonyms_words.csv]: waiting on feedback from Tracey for now but otherwise the PR looks good to me now.

Comment (lnazarenko) [reply training_data/en-GB/homonyms_words.csv]: filtered out Nazi_language subcategory from English rules

**Review by lsmith77 — APPROVED** [review]

**Pull request merged**